structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/converter/StructuredEventToCDPOperationDetailsConverter.java

```diff
@@ -27,8 +28,29 @@
         if (structuredFlowEvent == null) {
             return null;
         }
+        UsageProto.CDPOperationDetails.Builder cdpOperationDetails = convert(structuredFlowEvent.getOperation(), structuredFlowEvent.getStack(),
+                structuredFlowEvent.getFlow());
+
+        cdpOperationDetails.setCdpRequestProcessingStep(clusterRequestProcessingStepMapper.mapIt(structuredFlowEvent.getFlow()));
+
+        return cdpOperationDetails.build();
+    }
+
+    public UsageProto.CDPOperationDetails convert(StructuredSyncEvent structuredSyncEvent) {
+        if (structuredSyncEvent == null) {
+            return null;
+        }
+        UsageProto.CDPOperationDetails.Builder cdpOperationDetails = convert(structuredSyncEvent.getOperation(), structuredSyncEvent.getStack(), null);
+
+        cdpOperationDetails.setCdpRequestProcessingStep(UsageProto.CDPRequestProcessingStep.Value.UNSET);
```

Comment:
Can we have it as SYNC? It is cosmetic, but maybe it looks better.

Reply:
Changed it to SYNC.